zhmc_os_forwarder/schemas/config_schema.yaml

```diff
@@ -0,0 +1,87 @@
+# JSON Schema of the config file in YAML format.
+
+$schema: http://json-schema.org/draft-07/schema
+title: JSON Schema of the config file
+description: JSON Schema of the config file
+type: object
+required:
+  - hmc
+  - forwarding
+additionalProperties: false
+properties:
+  hmc:
+    description: The HMC the forwarder uses
+    type: object
+    required:
+      - hmc
+      - userid
+      - password
+    additionalProperties: false
+    properties:
+      hmc:
+        description: "Hostname or IP address of the HMC"
+        type: string
+      userid:
+        description: "Userid on the HMC"
+        type: string
+      password:
+        description: "Password of the HMC userid"
+        type: string
+      verify_cert:
+        description: "Controls whether and how the HMC certificate is verified: true, false, path name"
+        type: [boolean, string]
+  forwarding:
+    description: "Definition of forwarding items"
+    type: array
+    default: []
+    items:
+      description: "A forwarding item, that defines which partitions forward to which syslog servers"
+      type: object
+      required:
+        - syslogs
+        - cpcs
+      additionalProperties: false
+      properties:
+        syslogs:
+          description: "Syslog servers this forwarding item will forward to"
+          type: array
+          default: []
+          items:
+            description: "A syslog server"
+            type: object
+            required:
+              - server
+            additionalProperties: false
+            properties:
+              server:
```

Comment:
Should we allow to specify the port? Not sure whether all syslog servers are running on the default ports. Do we assume UDP or TCP as transport layer?

Reply:
We can add that as we learn about syslog forwarding. That's why I made the structure extensible.